ports/megamanx816-bit/megamanx816-bit/LICENSE.md

```diff
@@ -0,0 +1,22 @@
+X8 16-bit License
```

Comment:
We accept donations through the OpenCollective, doesnt that make the license invalid for PM?

Reply:
hmm fuck, i now wrote him on instagram and im hoping for an answer